From: David McCullough <david.mccullough@accelecon.com>
Date: Thu, 23 May 2013 14:50:19 +1000
Subject: build: binary initializer workaround

0b-prefixed binary initializers are not supported by some compilers.
Replaced binary initializer with hex.
---
 src/mm-iface-modem-3gpp-ussd.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/mm-iface-modem-3gpp-ussd.h b/src/mm-iface-modem-3gpp-ussd.h
index 37c60133..97cc618a 100644
--- a/src/mm-iface-modem-3gpp-ussd.h
+++ b/src/mm-iface-modem-3gpp-ussd.h
@@ -29,8 +29,8 @@
 #define MM_IFACE_MODEM_3GPP_USSD_GET_INTERFACE(obj) (G_TYPE_INSTANCE_GET_INTERFACE ((obj), MM_TYPE_IFACE_MODEM_3GPP_USSD, MMIfaceModem3gppUssd))
 
 /* CBS data coding scheme - 3GPP TS 23.038 */
-#define MM_MODEM_GSM_USSD_SCHEME_7BIT 0b00001111
-#define MM_MODEM_GSM_USSD_SCHEME_UCS2 0b01001000
+#define MM_MODEM_GSM_USSD_SCHEME_7BIT 0x0f //0b00001111
+#define MM_MODEM_GSM_USSD_SCHEME_UCS2 0x48 //0b01001000
 
 #define MM_IFACE_MODEM_3GPP_USSD_DBUS_SKELETON "iface-modem-3gpp-ussd-dbus-skeleton"
 
